android/app/src/main/java/org/uwaterloo/subletr/api/apis/ListingsApi.kt

```diff
@@ -125,6 +126,74 @@ class ListingsApi(basePath: kotlin.String = defaultBasePath, client: OkHttpClien
         )
     }
 
+    /**
+     * 
+     * 
+     * @return kotlin.String
+     * @throws IllegalStateException If the request is not correctly configured
+     * @throws IOException Rethrows the OkHttp execute method exception
+     * @throws UnsupportedOperationException If the API returns an informational or redirection response
+     * @throws ClientException If the API returns a client error response
+     * @throws ServerException If the API returns a server error response
+     */
+    @Suppress("UNCHECKED_CAST")
+    @Throws(IllegalStateException::class, IOException::class, UnsupportedOperationException::class, ClientException::class, ServerException::class)
+    suspend fun listingsDelete() : kotlin.String = withContext(Dispatchers.IO) {
```

Comment:
Just to confirm, this just deletes your listing, right?
If I can somewhat read Rust, I want to say yes....

Reply:
Yep!